com.unity.shadergraph/Editor/Data/Nodes/Math/Basic/MultiplyNode.cs

```diff
@@ -36,11 +36,9 @@ enum MultiplyType
 
         public override bool hasPreview => true;
 
-        string GetFunctionHeader()
+        string GetFunctionName()
         {
-            return "Unity_Multiply" + "_" + concretePrecision.ToShaderString()
-                + (this.GetSlots<DynamicVectorMaterialSlot>().Select(s => NodeUtils.GetSlotDimension(s.concreteValueType)).FirstOrDefault() ?? "")
-                + (this.GetSlots<DynamicMatrixMaterialSlot>().Select(s => NodeUtils.GetSlotDimension(s.concreteValueType)).FirstOrDefault() ?? "");
+            return $"Unity_Multiply_{FindSlot<MaterialSlot>(Input1SlotId).concreteValueType.ToShaderString()}_{FindSlot<MaterialSlot>(Input2SlotId).concreteValueType.ToShaderString()}";
```

Comment:
This was actually causing name collision errors, because it wasn't splicing the type names in properly.  Fixed it so we don't get name collisions, and it uses the precision tokens correctly for switchable precision.